cmd/gpu-kubelet-plugin/allocatable.go

```diff
@@ -66,9 +72,79 @@ func (d AllocatableDevice) UUID() string {
 	if d.Mig != nil {
 		return d.Mig.UUID
 	}
+	if d.Vfio != nil {
+		return d.Vfio.UUID
+	}
 	panic("unexpected type for AllocatableDevice")
 }
 
+type AllocatableDeviceList []*AllocatableDevice
+
+type AllocatableDevices map[string]*AllocatableDevice
+
+func (d AllocatableDevices) getDevicesByGPUPCIBusID(pcieBusID string) AllocatableDeviceList {
```

Comment:
Can we group the functions together that are only here because we dont yet support this feature via PartitionableDevices? Then have a comment before that block saying we can remove these once we support that.